cluster.go

```diff
@@ -223,12 +222,17 @@ func (c *Cluster) Validate(isTmpl bool) error {
 	return nil
 }
 
-func (c *Cluster) validateNode(n *Node, isTmpl bool, fldPath *field.Path) error {
-	if !isTmpl {
+func validateNode(n *Node, isTmpl bool, fldPath *field.Path) error {
+	if isTmpl {
+		if len(n.Address) != 0 {
+			return errors.New("address is not empty: " + n.Address)
+		}
```

Comment:
why do you check this?
anyways, the address will be overwritten.

Reply:
To check operation errors.

If the operator mistakenly set the CKE cluster definition (the result of `ckecli cluster get`) as a sabakan template, this check will cause an error.